tests/instantiation/test_configuration.py

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+from tests.parser.service_templates import consume_node_cellar
```

Comment:
You should use a custom example here rather than the nodecellar one IMO.
For once, these tests shouldn't break when the nodecellar example gets updated for some reason.
Another reason is that it's simply not comprehensive enough - there are two unit tests in this module, but a lot more scenarios to cover, for example:
  1) interface input which gets overridden by an operation input
  2) interface input which gets overriden by both an operation input and a configuration input
  3) interface input with a default value in the `interface_types` section without an override in the node template's interface section for that input
...

Even if we don't cover all of them, we should cover most, and this is not something you want to have as part of a single main example, but rather lots of small yaml sections each intended for a single purpose test.